src/transformers/integrations.py

```diff
@@ -668,16 +668,16 @@ def setup(self, args, state, model, **kwargs):
         variables:
 
         Environment:
-            WANDB_LOG_MODEL (`bool`, *optional*, defaults to `False`):
-                Whether or not to log model as artifact at the end of training. Use along with
-                *TrainingArguments.load_best_model_at_end* to upload best model.
-            WANDB_WATCH (`str`, *optional* defaults to `"gradients"`):
-                Can be `"gradients"`, `"all"` or `"false"`. Set to `"false"` to disable gradient logging or `"all"` to
-                log gradients and parameters.
-            WANDB_PROJECT (`str`, *optional*, defaults to `"huggingface"`):
-                Set this to a custom string to store results in a different project.
-            WANDB_DISABLED (`bool`, *optional*, defaults to `False`):
-                Whether or not to disable wandb entirely. Set *WANDB_DISABLED=true* to disable.
+        - WANDB_LOG_MODEL (`bool`, *optional*, defaults to `False`):
```

Comment:
```suggestion
        - **WANDB_LOG_MODEL** (`bool`, *optional*, defaults to `False`):
```
If we want to have the formatting as the arguments of a function/class, we can put all the names in bold.